src/main/java/frc/team2412/robot/commands/intake/IntakeInIdentifyBallCommand.java

```diff
@@ -2,7 +2,7 @@
 
 import frc.team2412.robot.subsystem.IntakeSubsystem;
 
-public class IntakeInIdentifyBallCommand extends IntakeInCommand {
+public class IntakeInIdentifyBallCommand extends IntakeMotorInCommand {
```

Comment:
What is this command being used for? To me it looks like `IntakeInIdentifyBallCommand` and `IntakeMotorInCommand` would have the same behavior.